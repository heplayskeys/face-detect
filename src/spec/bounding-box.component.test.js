import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import BoundingBox from '../components/bounding-box/bounding-box.component';

describe('<BoundingBox />', () => {
	it('returns correct bounding box snapshot with no "face" prop', () => {
		const face = { left_col: 0, right_col: 0, top_row: 0, bottom_row: 0 };
		const wrapper = shallow(<BoundingBox face={face} />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});
