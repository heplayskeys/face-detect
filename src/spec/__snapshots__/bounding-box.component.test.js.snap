// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoundingBox /> returns correct bounding box snapshot with no "face" prop 1`] = `
<div
  className="bounding-box grow animate__animated animate__fadeIn"
  style={
    Object {
      "bottom": NaN,
      "left": NaN,
      "right": NaN,
      "top": NaN,
    }
  }
/>
`;
